runtime/jcl/common/shared.c

```diff
@@ -1261,7 +1290,7 @@ Java_com_ibm_oti_shared_SharedClassURLHelperImpl_findSharedClassImpl3(JNIEnv* en
 	if (!getStringPair(env, &nameChars, &nameLen, &partitionChars, &partitionLen, classNameObj, partitionObj)) {
 		goto _errorPostClassNamePartition;
 	}
-	if (!(cpeType = getCpeTypeForProtocol((char*)urlElements.protocolChars, urlElements.protocolLen, urlElements.pathChars, urlElements.pathLen))) {
+	if (!(cpeType = getCpeTypeForProtocol(env, (char*)urlElements.protocolChars, urlElements.protocolLen, urlElements.pathChars, urlElements.pathLen))) {
```

Comment:
Assignment and test should be separate, Please compare with `0` (`CPE_TYPE_UNKNOWN`?) explicitly:
```c
	cpeType = getCpeTypeForProtocol(env, (char *)urlElements.protocolChars, urlElements.protocolLen, urlElements.pathChars, urlElements.pathLen));
	if (CPE_TYPE_UNKNOWN == cpeType) {
```